pkgs/top-level/all-packages.nix

```diff
@@ -11106,6 +11106,8 @@ with pkgs;
 
   wf-recorder = callPackage ../applications/video/wf-recorder { };
 
+  whatweb = callPackage ../tools/security/whatweb { };
```

Comment:
```suggestion
  whatweb = callPackage ../tools/security/whatweb { };
```
I don't see how this would be used for security.

Reply:
It is used for pentesting

Reply:
Also, it is listed at #81418.